docs/nuxt.config.js

```diff
@@ -0,0 +1,17 @@
+import theme from '@nuxt/content-theme-docs'
+
+export default theme({
+  generate: {
+    routes: ['/'],
+    fallback: '404.html'
```

Comment:
not related to this PR but I see fallback is set for all docs. Maybe better defaulting for theme preset? (which is still overridable)

Reply:
Full agree on this, cc @benjamincanac

Reply:
I agree too, is this just for Netlify or will it work for Vercel too?

Reply:
Not the fallback since it is configured to `200.html` by default.